articles/event-hubs/resource-governance-overview.md

```diff
@@ -67,14 +67,16 @@ The following table shows minimum threshold limits that you can set for differen
 > [!NOTE]
 > Limits set on the throttling policy's threshold value would take precedence over any value set for Kafka topic properties. For example, `IncomingBytes` would have higher priority over `message.max.bytes`.  
 
+> Application group throttling is expected to throttle consistent higher than permitted traffic scenarios (spanning across few minutes).Quick bursts in traffic for few seconds would not experience throttling via Application Groups. Looking at permitted throughput over the time horizon of few minutes is recommended approach to validate throttling.   
```

Comment:
Instead of "would not", we should say "may or may not" because if the spike is really huge, then we can respond in couple of secs.